sqlparse/lexer.py

```diff
@@ -187,6 +187,7 @@ class _Lexer(object):
             # IN is special, it may be followed by a parenthesis, but
             # is never a functino, see issue183
             (r'in\b(?=[ (])?', tokens.Keyword),
+            (r'USING(?=\()', is_keyword),
```

Comment:
this should probably be `(r'USING(?=\()', tokens.Keyword),` to skip going through the is_keyboard search.